solana/gatekeeper-lib/src/commands/blacklist.ts

```diff
@@ -0,0 +1,39 @@
+import { Command, Flags } from "@oclif/core";
```

Comment:
Please remove that file. Maybe add a comment with a reference to eth-ts that networks in eth support an authority abstraction.